src/classes/devices/difluidMicrobalance.ts

```diff
@@ -0,0 +1,172 @@
+import { PeripheralData } from './ble.types';
+import { BluetoothScale, SCALE_TIMER_COMMAND, Weight } from './bluetoothDevice';
+import { Logger } from './common/logger';
+
+declare var ble: any;
+export class DifluidMicrobalance extends BluetoothScale {
+  public static DEVICE_NAME = 'difluidmicrobalance';
+  public static SERVICE_UUID = '00EE';
+  public static CHAR_UUID = 'AA01';
+
+  protected override weight: Weight = {
+    actual: 0,
+    old: 0,
+    smoothed: 0,
+    oldSmoothed: 0,
+  };
+
+  private logger: Logger;
+  public scaleUnit = 'g';
+
+  constructor(data: PeripheralData) {
+    super(data);
+    this.logger = new Logger('DiFluid Scale');
+    this.connect();
+  }
+
+  public static test(device: any): boolean {
+    return (
+      device && device.name
+      // && this.DEVICE_NAME.includes(device.name)
+    );
+  }
+
+  public override async connect() {
+    this.logger.log('connecting...');
+    await this.attachNotification();
+    await this.setUnitToGram();
+    await this.enableAutoNotifications();
+  }
+
+  public override async tare() {
+    this.weight.smoothed = 0;
+    this.weight.actual = 0;
+    this.weight.oldSmoothed = 0;
+    this.weight.old = 0;
+    this.setWeight(0);
+
+    await this.write(
+      new Uint8Array([0xdf, 0xdf, 0x03, 0x02, 0x01, 0x01, 0xc5])
+    );
+  }
+
+  public override disconnectTriggered(): void {
+    this.logger.log('Disconnecting...');
+
+    this.deattachNotification();
+  }
+
+  public async setUnitToGram() {
+    await this.write(
+      new Uint8Array([0xdf, 0xdf, 0x01, 0x04, 0x01, 0x00, 0xc4])
+    );
+  }
+
+  public override async setTimer(_timer: SCALE_TIMER_COMMAND) {
+    this.logger.log('Setting Timer command ' + _timer + '...');
+
+    if (_timer === SCALE_TIMER_COMMAND.START) {
+      await this.write(
+        new Uint8Array([0xdf, 0xdf, 0x03, 0x02, 0x01, 0x00, 0xc4])
+      );
+    } else if (_timer === SCALE_TIMER_COMMAND.STOP) {
+      await this.write(
```

Comment:
What does this command? stop?

Reply:
I just simplified the timer management as the reset automatically happens anyway, if a new timer is started.